cmd/utils.go

```diff
@@ -142,3 +144,16 @@ func isNewer(c *clientContent, newerRef int) bool {
 	objectAge := UTCNow().Sub(c.Time)
 	return objectAge > (time.Duration(newerRef) * Day)
 }
+
+// getLookupType returns the minio.BucketLookupType for lookup
+// option entered on the command line
+func getLookupType(l string) minio.BucketLookupType {
+	l = strings.ToLower(l)
+	switch l {
+	case "dns":
+		return minio.BucketLookupDNS
```

Comment:
My 2 cents is that we should not have multiple returns, but a single return at the end of a function, like:
```
func getLookupType(l string) minio.BucketLookupType {
   l = strings.ToLower(l)
   var lookupVal minio.BucketLookupType
   switch l {
      case "dns":
         lookupVal = minio.BucketLookupDNS
      case "path":
         lookupVal = minio.BucketLookupPath
      default:
         lookupVal = minio.BucketLookupAuto
      }
   return lookupVal
}
```